### ISSUE TITLE
not working for mobile device or IOS

### ISSUE THREAD

**Issue by ankesh-m** [alexk111/ngImgCrop#117]
When i am using the plugin it is working fine in desktop but when i am using the same in mobile device or ios then it will not work

**Comment by ankesh-m** [comment]
hi @alexk111 any idea about this issue?

**Comment by ketansp** [comment]
Its working in my project. At least on chrome in android.

**Comment by outlawbruce** [comment]
Hi ketansp did you try and see if it works on any of your samsung devices? Im having issues with a few weeks now.

**Comment by ketansp** [comment]
Hi outlawbruce,

I have tested it on a variety of Android devices. Works well with all of
them. I will arrange to test this on a Samsung device and confirm the same.

Regards,

Ketan Patil.
-- sent from a mobile device

**Comment by eriksendc** [comment]
Hi @ankesh-m (and all),

I noticed something today that I think relates to this thread. It's not really iOS per-se, but photos created by iOS perhaps? Anyways, result-image-format defaults to image/png, I believe. When I use it like that then photos taken on my iPhone don't appear when I select them. But when I change result-image-format to image/jpeg then things work as expected.

Another thing to note is that the main public demo jsfiddle suffers from this issue, whether you use Safari or Chrome on an iPhone. If I go to jsfiddle.net/alexk111/rw6q9/ on my iPhone I do not see my images in the preview or crop divs after selecting them.

I don't have my code running somewhere where I can quickly point my iPhone to it when running locally off my laptop. I'll write another note on this thread when I have more info. But for now it seems like there's something weird with iPhone

-Brian Eriksen

**Comment by jgoslow** [comment]
I can provide some more feedback on this - I'm trying to use this plugin for a project, and after doing some testing on my iphone I found that the plugin broke if the images I uploaded were taken in a certain orientation (meaning my phone was rotated a certain way).  

iOS stores orientation in what's called EXIF data, and if the orientation is anything but the standard (landscape with the charge plug on the left), then it breaks the plugin.  Can anyone corroborate this?

**Comment by dazulu** [comment]
@jgoslow Yes! Android tablet, Android phone, Macbook Pro, and Win10 laptop all fine. However, when I select a photo that has been taken with an iPhone (5 and 6s) either via file selector or taking the photo directly, it doesn't load. I've had limited success with taking a landscape photo but it really needs to be perfect landscape orientation. It does work with photos transferred to the phone.

**Comment by jgoslow** [comment]
There's another comment thread that answers this problem @dazulu [here](https://github.com/alexk111/ngImgCrop/issues/58), and if you want to just get the gist file, that is [here](https://gist.github.com/hbbh/d5f21c4120665f9429e3).

**Comment by dazulu** [comment]
Thanks for this!

Sent from Outlook Mobile




On Sat, Feb 27, 2016 at 9:01 AM -0800, "Jonas Goslow" <notifications@github.com> wrote:





There's another comment thread that answers this problem @dazulu [here](https://github.com/alexk111/ngImgCrop/issues/58), and if you want to just get the gist file, that is [here](https://gist.github.com/hbbh/d5f21c4120665f9429e3).

---
